R/addAlpha.R

```diff
@@ -5,46 +5,46 @@
 #' 
 #' @param x a \code{\link{SummarizedExperiment}} object.
 #' 
-#' @param assay.type the name of the assay used for calculation of the
-#'   sample-wise estimates. (Default: \code{"counts"})
+#' @param assay.type \code{Character vector}. Specifies the name of assay 
+#'   used in calculation. (Default: \code{"counts"})
 #'   
-#' @param index a \code{character} vector, specifying the alpha diversity 
+#' @param index \code{Character vector}. Specifies the alpha diversity 
 #'   indices to be calculated.
 #'   
-#' @param name a name for the column(s) of the colData the results should be
+#' @param name \code{Character scalar}. A name for the column(s) of the colData the 
+#'   results should be
 #'   stored in. By default this will use the original names of the calculated
 #'   indices. (Default: \code{index})
 #' 
-#' @param niter \code{NULL} or a single \code{integer} value for the number of
+#' @param niter \code{Integer scalar}. Specifies the number of
 #'   rarefaction rounds. Rarefaction is not applied when \code{niter=NULL}
 #'   (see Details section). (Default: \code{NULL})
 #'   
 #' @param ... optional arguments:
 #' \itemize{
-#'   \item \code{sample}: A \code{numeric} value specifying the rarefaction
+#'   \item \code{sample}: \code{Numeric scalar}. Specifies the rarefaction
 #'   depth i.e. the number of counts drawn from each sample.
 #'   (Default: \code{min(colSums2(assay(x, assay.type)))})
 #'   
-#'   \item \code{tree.name}: (Faith's index) A single \code{character}
-#'   value for specifying which rowTree will be used. 
-#'   (Default: \code{"phylo"})
+#'   \item \code{tree.name}: \code{Character scalar}.Specifies which rowTree 
+#'    will be used. ( Faith's index). (Default: \code{"phylo"})
 #'   
-#'   \item \code{node.label}: (Faith's index) \code{NULL} or a
-#'   \code{character} vector specifying the links between rows and node labels
-#'   of phylogeny tree specified by \code{tree.name}. If a certain row is not
-#'   linked with the tree, missing instance should be noted as NA. When
-#'   \code{NULL}, all the rownames should be found from the tree.
-#'   (Default: \code{NULL})
+#'   \item \code{node.label}: \code{Character vector} or \code{NULL} Specifies the 
+#'   links between rows and node labels of phylogeny tree specified 
+#'   by \code{tree.name}. If a certain row is not linked with the tree, missing 
+#'   instance should be noted as NA. When \code{NULL}, all the rownames should 
+#'   be found from the tree. (Faith's index). (Default: \code{NULL})
 #'   
-#'   \item \code{only.tips}: (Faith's index) A \code{boolean} value
-#'   specifying whether to remove internal nodes when Faith's index is
+#'   \item \code{only.tips}: \code{Logical scalar}. Specifies
+#'   whether to remove internal nodes when Faith's index is
 #'   calculated. When \code{only.tips=TRUE}, those rows that are not tips of
-#'   tree are removed. (Default: \code{FALSE})
```

Comment:
As these are affecting just certain indices, it might be better to say it first. (That is the most important information, then comes the class, then the detailed description)